src/index.ts

```diff
@@ -579,6 +579,40 @@ function steemAddressDecoder(data: string): Buffer {
   return Buffer.from(key);
 }
 
+// Referenced from following
+// https://dev.bitshares.works/en/master/bts_guide/index_faq.html#dev-faq17
+function btsAddressEncoder(data: Buffer): string {  
+  const RIPEMD160 = require('ripemd160');
```

Comment:
Please use `import` at the beginning of the file rather than requiring on the spot

Reply:
Sure, on it